src/praisonai-deploy/praisonai_deploy/main.py

```
@@ -99,28 +100,35 @@ def _deploy_api(self, background: bool = False) -> DeployResult:
-        
-        # Build image
-        build_result = build_docker_image(self.config.docker)
-        
+        from .models import APIConfig
+
+        _, agents_basename, _ = resolve_docker_paths(self.agents_file)
+        docker_cfg = self.config.docker
+        assert docker_cfg is not None
+
+        api_config = self.config.api or APIConfig(
+            host="0.0.0.0",
```

Ruff (0.16.0)

[error] Possible binding to all interfaces

(S104)